### ISSUE TITLE
Intercom API should take a method name and an object, not array.

### ISSUE THREAD

**Issue by jonathan3692bf** [nhagen/react-intercom#86]
In [src/intercomAPI.ts](https://github.com/nhagen/react-intercom/blob/bb9c20269ea61a28a1065c46badeb6a8c73d64ea/src/intercomAPI.ts#L3) on branch `2.0`, `args` should be an `object`, not an `array`.

Just change line 1 from:
```
export default function(method: string, ...args: Array<any>) {
```
to:
```
export default function(method: string, args: any) {
```

**Comment by ChristianJacobsen** [comment]
Not sure if this is what is being proposed fixed in #77? I think it is.

**Comment by jonathan3692bf** [comment]
True, though #77 lumps `method` and `args` together. Either way, we achieve the same goal, so I'll close this issue.
